app/controllers/cart_items_controller.rb

```diff
@@ -1,8 +1,12 @@
 class CartItemsController < ApplicationController
+  include CurrentCart
```

Comment:
you don't need this here because you already included this module in application controller